src/javascript/app/ng-wig/ng-wig-toolbar.provider.spec.js

```diff
@@ -5,126 +5,128 @@ describe('provider: ngWigToolbar', function () {
         module('ngWig', (_ngWigToolbarProvider_) => {
             toolbarProvider = _ngWigToolbarProvider_;
         });
+
+        inject();
```

Comment:
Generally yes. If we are testing components/services a simple call like the following is sufficient:

`beforeEach(module('ngWig'));`

But in the case of providers, where you may have to configure it before loading the main module, you define the module first and then inject it. Have a look in [this](hhttps://github.com/johnpapa/ng-demos/blob/master/modular/src/client/test/specs/exception-handler.provider.spec.js#L10) exception handler provider John Papa.